### PR TITLE
Add Rocket icon

### LINKED ISSUE #8185: Add Rocket icon
### Brand Name

Rocket

### Website

https://rocket.rs

### Popularity Metric

467,655 on similarweb - https://www.similarweb.com/website/rocket.rs
19,376 stars on GitHub - https://github.com/SergioBenitez/Rocket

### Official Resources for Icon and Color

boxed png: https://rocket.rs/images/logo-boxed.png
rocket svg: https://rocket.rs/images/logo-small.svg
color: `#d33847` (from logo)

### Additional Comments

_No response_

### Contributing

No

### PR DESCRIPTION
**Pull request by IamRezaMousavi**
rocket

**Issue:** closes #8185

 [21.4k GitHub ⭐](https://github.com/SergioBenitez/Rocket)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Did you manually vectorize the logo? No

### CODE REVIEW

**Comment by adamrusted** [comment]
Hey @SergioBenitez - you wouldn't happen to have an SVG/AI version of the icon shown above would you?
It'd be ace to include the original icon as opposed to one of our contributors having to manually vectorize it!

**Comment by IamRezaMousavi** [comment]
No, The site svg icon is different
This svg is only used for the header and footer of the site:
logo-small

Origin icon which used in head of origin html page is this:
logo-boxed

**Comment by adamrusted** [comment]
I don't see the version with the trails behind the rocket used in the current source code of the homepage. We might just need to use the boxed rocket without the trails.

**Comment by SergioBenitez** [comment]
Here's everything:

logo-small
logo
logo-boxed
favicon
rocket-logo@2x
rocket-logo@3x

**Comment by IamRezaMousavi** [comment]
@adamrusted What do I need to do?

**Comment by PeterShaggyNoble** [comment]
@IamRezaMousavi, can you redo your SVG based on [this one](https://user-images.githubusercontent.com/1480321/269096120-63020ac0-9ad6-4a1b-851b-327019a9bdb0.svg) provided above, please?

**Comment by IamRezaMousavi** [comment]
> @IamRezaMousavi, can you redo your SVG based on [this one](https://user-images.githubusercontent.com/1480321/269096120-63020ac0-9ad6-4a1b-851b-327019a9bdb0.svg) provided above, please?

@PeterShaggyNoble How is it now?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11204,6 +11204,11 @@
                 "type": "CC-BY-NC-SA-4.0"
             }
         },
+        {
+            "title": "Rocket",
+            "hex": "D33847",
+            "source": "https://rocket.rs/images/logo-boxed.png"
```
Comment (adamrusted) [inline _data/simple-icons.json:11210]:
```suggestion
            "source": "https://github.com/simple-icons/simple-icons/pull/9492#issuecomment-1726472160"
```

**Review by PeterShaggyNoble — CHANGES_REQUESTED** [review]
Sorry, @IamRezaMousavi, I should have been clearer: it's that version we should be using but with the square. Like so:
image

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @IamRezaMousavi?

**Comment by IamRezaMousavi** [comment]
> Are you still working on this one, @IamRezaMousavi?

I think Rocket is not qualified for [your new review metrics](https://github.com/simple-icons/simple-icons/issues/9979)

Please see [similarweb ranking](https://www.similarweb.com/website/rocket.rs/#overview)

**Comment by adamrusted** [comment]
> I think Rocket is not qualified for your new review metrics

SimilarWeb rank of 306,696 puts this in scope!

**Comment by adamrusted** [comment]
Given it's still in scope - are you able to make the changes requested, @IamRezaMousavi?

**Comment by IamRezaMousavi** [comment]
> Sorry, @IamRezaMousavi, I should have been clearer: it's that version we should be using but but with the square. Like so: image

I think you wrong about this

**Comment by IamRezaMousavi** [comment]
I redid the logo based on [this one](https://user-images.githubusercontent.com/1480321/269096120-63020ac0-9ad6-4a1b-851b-327019a9bdb0.svg)

File: icons/rocket.svg
Comment (PratyushJayachandran) [inline icons/rocket.svg:1]: There is a slight bit of scaling issue which could also be due to SVG optimization. [See comparison.](https://github.com/simple-icons/simple-icons/assets/35000807/8f923197-8abe-496a-896c-38f8d9a60db1)
If it still doesn't improve try upping the precision to 4.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -12845,6 +12845,11 @@
                 "type": "CC-BY-NC-SA-4.0"
             }
         },
+        {
+            "title": "Rocket",
+            "hex": "D33847",
+            "source": "https://github.com/simple-icons/simple-icons/pull/9492#issuecomment-1726472160"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:12851]: We need an actual source from where the logo was picked up. Not a comment from the PR itself. Well if the author or someone had provided in the PR comments, that would have been acceptable.

Anyway, how about we use the logo-small.svg from their own website itself?
```suggestion
            "source": "https://rocket.rs/"
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Can you please re-do from the logo-small.svg from their official website?
Thank you for your persistent efforts @IamRezaMousavi 😄

File: icons/rocket.svg
Comment (PratyushJayachandran) [reply icons/rocket.svg:1]: Let's use the new source instead: https://rocket.rs/images/logo-small.svg

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran, we need to use one of [these SVGs](https://github.com/simple-icons/simple-icons/pull/9492#issuecomment-1726472160) as they have been provided to us by an official source. I opted for the one with square corners and without the trails as it would look better at our native size.

I compared the one posted by the creator and the one on the latest website. The alignment of the fit inside the box is different for both. In the interest of not dragging this further, I will apply changes so that you can review and merge.

The changes are based on SVG on the website header itself: https://rocket.rs/images/logo-small.svg

File: icons/rocket.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rocket</title><path d="M3.006 10.936c-1.068 1.132-2 2.39-3.001 3.573-.076.62.8.517 1.214.72l2.837.79c.394-.488 1.349-1.49 1.546-1.495-.319.873-1.564 1.418-1.305 2.418.804.989.402 1.993-.604 2.628-.584.67.853 1.514 1.38 1.835.888-.812 1.635-1.94 2.974-1.332 1.284.238 2.026-1.04 3.128-1.414-.405.654-1.523 1.041-1.642 1.687l1.375 3.623c.768.224 1.19-.827 1.783-1.225l1.918-1.836c.389-1.398.443-2.87.826-4.265.985-.975 2.036-1.953 2.945-3.03 2.815-3.251 5.217-7.176 5.564-11.557.18-1.054-.026-2.312-1.388-2.01-5.093.394-9.565 3.414-13.08 6.952-.912.773-1.51 1.82-2.342 2.659-.714.423-1.586.504-2.369.778zm9.89-3.175c1.047-1.15 3.173-1.749 4.315-.412 1.073 1.563-.02 3.666-1.488 4.532-1.144.827-3.168.818-3.797-.644-.499-1.216.093-2.6.97-3.476"/></svg>
```
Comment (PratyushJayachandran) [inline icons/rocket.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rocket</title><path d="M23.737.237V.236a.247.247 0 0 0-.201-.188C23.28.007 17.2-.877 9.366 7.1l-.001.001c-.039.041-1.043 1.101-2.118 2.575l-4.049 1.166a.246.246 0 0 0-.123.08l-2.856 3.5a.247.247 0 0 0 .124.393l3.89 1.096.483-.565c.177-.208.37-.401.575-.58l.54-.471-.38.607c-.145.231-.307.451-.483.658l-.519.607c.007.79.213 1.489.62 2.069L3.68 19.653c-.147.162-.037.272.122.429l1.108 1.086c.153.16.257.258.41.099l1.505-1.533c.343.122 1.163.333 2.4.14l.75-.576a5.55 5.55 0 0 1 .673-.442l.644-.36-.514.53a5.549 5.549 0 0 1-.601.534l-.62.476 1.425 3.804a.246.246 0 0 0 .403.09l3.243-3.143a.248.248 0 0 0 .072-.136l.697-4.11c.885-.78 1.78-1.686 2.661-2.694 5.072-5.805 5.797-10.315 5.778-12.486-.006-.703-.093-1.095-.1-1.124zm-7.245 10.928c-1.274 1.296-3.101 1.566-4.082.602-.98-.964-.743-2.795.53-4.091 1.274-1.296 3.101-1.566 4.082-.602.98.964.743 2.795-.53 4.091z"/></svg>
```

File: icons/rocket.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rocket</title><path d="M23.737.237V.236a.247.247 0 0 0-.201-.188C23.28.007 17.2-.877 9.366 7.1l-.001.001c-.039.041-1.043 1.101-2.118 2.575l-4.049 1.166a.246.246 0 0 0-.123.08l-2.856 3.5a.247.247 0 0 0 .124.393l3.89 1.096.483-.565c.177-.208.37-.401.575-.58l.54-.471-.38.607c-.145.231-.307.451-.483.658l-.519.607c.007.79.213 1.489.62 2.069L3.68 19.653c-.147.162-.037.272.122.429l1.108 1.086c.153.16.257.258.41.099l1.505-1.533c.343.122 1.163.333 2.4.14l.75-.576a5.55 5.55 0 0 1 .673-.442l.644-.36-.514.53a5.549 5.549 0 0 1-.601.534l-.62.476 1.425 3.804a.246.246 0 0 0 .403.09l3.243-3.143a.248.248 0 0 0 .072-.136l.697-4.11c.885-.78 1.78-1.686 2.661-2.694 5.072-5.805 5.797-10.315 5.778-12.486-.006-.703-.093-1.095-.1-1.124zm-7.245 10.928c-1.274 1.296-3.101 1.566-4.082.602-.98-.964-.743-2.795.53-4.091 1.274-1.296 3.101-1.566 4.082-.602.98.964.743 2.795-.53 4.091z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/rocket.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Rocket</title><path d="M23.735.238V.236a.248.248 0 0 0-.2-.188c-.256-.04-6.336-.924-14.17 7.051a28.44 28.44 0 0 0-2.12 2.576l-4.047 1.166a.246.246 0 0 0-.124.08l-2.856 3.5a.248.248 0 0 0 .126.394l3.887 1.096.484-.566c.178-.208.37-.4.574-.58l.54-.472-.38.608a5.556 5.556 0 0 1-.482.66l-.52.606c.008.79.214 1.488.62 2.068L3.68 19.653c-.148.16-.036.272.12.428l1.11 1.086c.153.16.255.258.41.1l1.505-1.534c.34.122 1.162.334 2.4.14l.75-.576c.212-.164.438-.312.672-.442l.644-.36-.514.53c-.187.192-.387.37-.6.534l-.62.476 1.424 3.804a.246.246 0 0 0 .404.09l3.242-3.144a.248.248 0 0 0 .072-.136l.698-4.108c.884-.78 1.78-1.686 2.66-2.694 5.072-5.806 5.798-10.315 5.78-12.487-.008-.702-.094-1.094-.1-1.122h-.002zM16.49 11.165c-1.274 1.296-3.1 1.564-4.082.6-.98-.962-.744-2.794.53-4.09s3.1-1.566 4.08-.602c.982.964.746 2.796-.528 4.092z"/></svg>
```

**Review by PratyushJayachandran — APPROVED** [review]
Matches perfectly @IamRezaMousavi 🎉 
Also thanks to @adamrusted and @PeterShaggyNoble for continued discussions and review!

**Pull request merged**